sematic/api/endpoints/resolutions.py

```diff
@@ -83,7 +91,12 @@ def put_resolution_endpoint(user: Optional[User], resolution_id: str) -> flask.R
 
     try:
         if existing_resolution is not None:
-            existing_resolution.update_with(resolution)
+            existing_resolution.update_with(
+                resolution,
+                # This field is scrubbed on read, but should be immutable.
+                # ignore whatever the caller sent back this time.
+                ignore_conflicts=[Resolution.settings_env_vars.key]
```

Comment:
Instead of this, why not just set `resolution.settings_env_vars` to `existing_resolution.settings_env_vats` before passing to `update_with`?

Reply:
because that would be too obvious 😆